### ISSUE TITLE
When to expect v2.0?

### ISSUE THREAD

**Issue by LifeImprovingInnovations** [tmoroney/auto-subs#100]

**Comment by GabrielM33** [comment]
Also waiting on it

**Comment by tmoroney** [comment]
In a few days for Mac, and probably a week for Windows. I need to work out how to sign windows applications so they don't trigger people's anti-virus.

**Comment by LifeImprovingInnovations** [comment]
I'm waiting for the Windows version. Hopefully will come soon!!!

**Comment by tmoroney** [comment]
This issue was opened against an older version of AutoSubs. The project has been completely rewritten since then — please update to the latest version as this is very likely already fixed. If you're still experiencing this on the latest version, please open a new issue with updated logs.

---
_Generated by [Claude Code](https://claude.ai/code)_

**Comment by tmoroney** [comment]
This issue was opened against an older version of AutoSubs. The project has been completely rewritten since then — please update to the latest version as this is very likely already fixed. If you're still experiencing this on the latest version, please open a new issue with updated logs.

---
_Generated by [Claude Code](https://claude.ai/code)_
